src/components/shared/DropdownField/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+type DropdownFieldProps = {
+  id: string;
+  label?: string;
+  children: React.ReactNodeArray;
+};
+export const DropdownField = ({ id, label, children }: DropdownFieldProps) => (
+  <>
+    {label && (
+      <label className="usa-label" htmlFor={id}>
+        {label}
+      </label>
+    )}
+    <select className="usa-select" name={id} id={id}>
+      {children}
+    </select>
+  </>
+);
+
+type DropdownItemProps = {
+  children: React.ReactNode;
+  value: string;
+};
+
+export const DropdownItem = ({ children, value }: DropdownItemProps) => (
```

Comment:
Could the props for this be more descriptive, like value and label? Will the children ever be anything besides a string?

Reply:
`this.props.value` maps 1:1 to the HTML attribute `value`.
Since these are generic components, I wanted the names to be as generic as possible and map to the HTML attributes.

As for `children`, 99.9% of the time it's going to be a `string`. We can pass in the label as a prop too. I don't mind changing it.

Reply:
Actually, I stand corrected. HTML elements cannot be nested in `option` tags. Therefore, this should always be a string.